src/lib/notifications.js

```diff
@@ -70,6 +73,7 @@ class PushNotification extends Notification {
                   closeOnClick: true,
                   icon: 'img/icons/android-chrome-192x192.png',
                   notifyClick: () => {
+                    this.increaseCounter()
```

Comment:
We increase counter as workaround to easily detect need of scrolling to bottom in currently opened component